src/lib.rs

```diff
@@ -60,9 +60,7 @@ macro_rules! kernel_module {
         #[link_section = ".modinfo"]
         #[allow(non_upper_case_globals)]
         // TODO: Generate a name the same way the kernel's `__MODULE_INFO` does.
-        // TODO: This needs to be a `&'static [u8]`, since the kernel defines this as a
```

Comment:
So on further consideration this isn't right -- though it is closer to right! We need the value to be a `[u8; _]`, not a `&[u8]`, that is, no pointer. Can you update the comment, but leave the actual code change, since this is forward progress?

Reply:
I'm not really sure what you mean by:
> Can you update the comment, but leave the actual code change

What do you want the comment to say? And is the underscore in `[u8; _]` supported yet? Last time I checked it wasn't so I don't think we have a way to have a compiler determined width array?

Reply:
I'd change the line to "TODO: This needs to be a `[u8; _]`, since the kernel defines this as a". The code change can stay